TwitchLib.Api.Helix.Models/Webhooks/GetWebhookSubscriptionsResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+using TwitchLib.Api.Helix.Models.Common;
+
+namespace TwitchLib.Api.Helix.Models.Webhooks
+{
+    public class GetWebhookSubscriptionsResponse
```

Comment:
Might be best to add summaries to publicly available models, easier to add them now than to add them later.